examples/tutorials/date_time_charts.py

```diff
@@ -0,0 +1,351 @@
+"""
+Plotting datetime charts
+========================
+
+PyGMT accepts a variety of datetime objects to plot data and create charts.
+Aside from the built-in Python datetime object, PyGMT supports input using
+properly formatted strings, ``pandas``, ``xarray``, as well as ``numpy``.
```

Comment:
In this tutorial, formatted strings mean ISO format? If so, shall we explicitly indicate it here?
```suggestion
properly formatted strings, ``pandas``, ``xarray``, as well as ``numpy``.
```